libs/barista-components/experimental/quick-filter/src/quick-filter.html

```diff
@@ -1,6 +1,8 @@
 <dt-filter-field
   [dataSource]="_filterFieldDataSource$ | async"
-  (filterChanges)="_filterFiledChanged($event)"
+  (inputChange)="inputChange.emit($event)"
+  (currentFilterChanges)="currentFilterChanges.emit($event)"
+  (filterChanges)="_filterFieldChanged($event)"
```

Comment:
@thomaspink I think there is a better way to proxy the event through, without emitting a new one. 
Can you please advise?